pennylane_lightning_gpu/src/simulator/StateVectorCudaManaged.hpp

```diff
@@ -1092,7 +1091,19 @@ class StateVectorCudaManaged
         }
     }
 
+    auto getCublasHandle() const -> const CublasCaller & {
+        return *cublashandle;
+    }
+    auto getCusparseHandle() const -> cusparseHandle_t {
+        if (!cusparsehandle)
+            cusparsehandle = make_shared_cusparse_handle();
+        return cusparsehandle.get();
+    }
```

Comment:
Since this is public API facing, a docstring here would be appreciated.

Reply:
done.